src/data/area_registry.ts

```diff
@@ -58,7 +58,7 @@ const fetchAreaRegistry = (conn: Connection) =>
     })
     .then((areas) =>
       (areas as AreaRegistryEntry[]).sort((ent1, ent2) =>
-        stringCompare(ent1.name, ent2.name)
+        stringCompare(ent1.name, ent2.name, "en")
```

Comment:
Don't use hardcoded languages, if undefined, the browser lang will be used.

Reply:
Done.